### DISCUSSION TITLE
New website

### DISCUSSION THREAD

**Discussion by mickem** [mickem/nscp#783] [Announcements]
To make life easier I converted most of the web site to "mk-docs" and merged it with the docs site.
This means updating the docs is "free" as in no effort from my side which is great :)

The drawback is that I have removed some features:
* the mostly defunct crash reporting site (might re-add at a later time if there is interest)
* The "old" forums
* The "download" section, which was really just a fancy UI for github actions
* Archive of old documentations (can still be downloaded for offline use from download archive)

Please let me know if you run into any issues or if something is rendering wrongly or you find broken links...

// Michael Medin

**Comment by jadzdotcom** [comment]
I was thrown off by the new site, thinking that I was looking at unofficial documentation and even went to archive.org to confirm that the site was different last year (i.e. a few days ago). All the things you are doing to make this project more easily maintained make a lot of sense to me. 

One thing I noticed is that Google's search results are currently broken (i.e. searching for `nsclient check_logfile` results in https://docs.nsclient.org/reference/check/CheckLogFile/ which just redirects you to the main documentation page, instead of the new URL which is https://nsclient.org/docs/reference/check/CheckLogFile/ ). I assume this will fix itself after the site gets re-indexed.
